src/pages/createProfile.js

```diff
@@ -167,7 +179,7 @@ export default function CreateProfile() {
                   <TextField
                     required
                     margin="normal"
-                    placeholder="Location"
+                    placeholder="Current City"
```

Comment:
The field is missing implementation that we discussed where the user gets recommendations of where the state and city is.